dds/DCPS/transport/tcp/TcpDataLink.cpp

```diff
@@ -331,14 +331,107 @@ OpenDDS::DCPS::TcpDataLink::send_graceful_disconnect_message()
   this->send_i(send_element, false);
 }
 
-void OpenDDS::DCPS::TcpDataLink::set_release_pending(bool flag)
+void
+OpenDDS::DCPS::TcpDataLink::set_release_pending(bool flag)
 {
   this->release_is_pending_ = flag;
 }
 
-bool OpenDDS::DCPS::TcpDataLink::is_release_pending() const
+bool
+OpenDDS::DCPS::TcpDataLink::is_release_pending() const
 {
   return this->release_is_pending_.value();
 }
 
+bool
+OpenDDS::DCPS::TcpDataLink::handle_send_request_ack(TransportQueueElement* element)
+{
+  if (DCPS_debug_level >= 1) {
```

Comment:
This and other instances within dds/DCPS/transport/* should use the transport debug level instead of DCPS's debug level.